### PR TITLE
[CU-302tgzn] Add a building of composable-centauri docker image

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/302tgzn

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -616,6 +649,14 @@
               chain-spec = "dali-dev";
             }).script;
 
+            # Dali Centauri devnet
+            devnet-dali-centauri = (callPackage mk-devnet {
```
Comment (dzmitry-lahoda) [inline flake.nix:653]: can we prefix this like

bridge-devnet-dali

File: flake.nix [hunk 2]
```diff
@@ -644,6 +685,28 @@
               '';
             };
 
+            # Dali Centauri devnet container
+            centauri-devnet-container = dockerTools.buildImage {
```
Comment (dzmitry-lahoda) [inline flake.nix:689]: bridge-devnet-dali

**Review by dzmitry-lahoda — COMMENTED** [review]
I think we should prefix temporal forks by team names which requested that. centauri prefix is okey too. so I would prefer bridge.

File: flake.nix (hunk 1 above)
Comment (andor0) [reply flake.nix]: fixed

File: flake.nix (hunk 2 above)
Comment (andor0) [reply flake.nix]: fixed

**Comment by andor0** [comment]
> I think we should prefix temporal forks by team names which requested that. centauri prefix is okey too. so I would prefer bridge.

I added `bridge` prefix.

**Review by haroldsphinx — APPROVED** [review]

**Review by blasrodri — APPROVED** [review]

**Pull request merged**